internal/service/remotecfg/remotecfg.go

```diff
@@ -591,3 +591,16 @@ func (i *agentInterceptor) WrapStreamingClient(next connect.StreamingClientFunc)
 func (i *agentInterceptor) WrapStreamingHandler(next connect.StreamingHandlerFunc) connect.StreamingHandlerFunc {
 	return next
 }
+
+func GetRemoteCfgHost(host service.Host) (service.Host, error) {
```

Comment:
nit: consider naming `GetHost` as the remotecfg part is implicit from the package.